user_templates/mcu_defaults/main/BTT_SKR_Pico_v1.0.cfg

```diff
@@ -0,0 +1,34 @@
+
+#----------------------------------------#
+#### BTT SKR Pico v1.0 definition ########
+#----------------------------------------#
+
+[mcu]
+##--------------------------------------------------------------------
+serial: /dev/serial/by-id/change-me-to-the-correct-mcu-path
+##--------------------------------------------------------------------
+
+[include config/mcu_definitions/main/BTT_SKR_Pico_v1.0.cfg] # Do not remove this line
+[board_pins xye_SKR_mcu]
+mcu: mcu
+aliases:
+    X_STEP=MCU_XM_STEP , X_DIR=MCU_XM_DIR , X_ENABLE=MCU_XM_ENABLE ,
+    Y_STEP=MCU_YM_STEP , Y_DIR=MCU_YM_DIR , Y_ENABLE=MCU_YM_ENABLE ,
+    Z_STEP=MCU_ZM_STEP , Z_DIR=MCU_ZM_DIR , Z_ENABLE=MCU_ZM_ENABLE ,
+    E_STEP=MCU_EM_STEP , E_DIR=MCU_EM_DIR , E_ENABLE=MCU_EM_ENABLE ,
+
+    XYZE_TMCUART=MCU_TMCUART , XYZE_TMCTX=MCU_TMCTX , 
+
+    X_STOP=MCU_XSTOP , Y_STOP=MCU_YSTOP , Z_STOP=MCU_ZSTOP ,
+    RUNOUT_SENSOR=MCU_E0STOP ,
+    PROBE_INPUT=MCU_PROBE    ,
+
+    E_HEATER=MCU_HE0   , E_TEMPERATURE=MCU_TH0  ,
+    BED_HEATER=MCU_BED , BED_TEMPERATURE=MCU_TB ,
+
+    PART_FAN=MCU_FAN1 , E_FAN=MCU_FAN2 ,
+    CONTROLLER_FAN=MCU_FAN3 ,
+
+
+    STATUS_NEOPIXEL=MCU_NEOPIXEL ,
+    LIGHT_NEOPIXEL=MCU_NEOPIXEL  ,
```

Comment:
So this should be changed accordingly to follow the manufacturer file.

Also is the neopixel double definition working? If this is not the case, I suggest commenting one of them with a note like: "Since this MCU have only one neopixel port, uncomment the one used in your config. You cannot use both at the same time."